packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -211,6 +211,7 @@ class SnackBar extends StatefulWidget {
     this.padding,
     this.width,
     this.shape,
+    this.hitBehavior = HitTestBehavior.translucent,
```

Comment:
I think the name should be `hitTestBehavior` for consistency with other similar parameters in Flutter.

Reply:
Yes, this is intentional, because with `transcluent` we have no problems with gestures and taps under the snackbar.
In the above issues, `Dismissible`'s default `behavior` is `opaque`

Agree about the name, I'll change it now